config/development_mysql.yaml

```diff
@@ -85,6 +89,8 @@ archival:
       filestore:
         fileMode: "0666"
         dirMode: "0766"
+      gstorage:
```

Comment:
Do we need this?

Reply:
You know better. I found it in one the configs and just copied it everywhere. Should it be in `development_cass-archival.yaml` only?

Reply:
My understanding is the dev config for file store and it will ignore the gstorage config. @yycptt Do you have context here?

Reply:
I think we should keep it. Although the default URI is local file store, the gstorage section may still be used if someone register a namespace with gcp archival URI in local environment for development purpose.